requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
+pytest
+pytest-cov
+pytest-pep8
```

Comment:
We might want to list `sphinx` and `sphinx_rtd_theme` as well. Developers will usually need that to ensure they don't commit erroneous markup.